docs/migrate-module.md

```diff
@@ -0,0 +1,100 @@
+# Update an older module
+
+Modules in the IBM Terraform modules project need specific logic and workflows to pass the continuous integration (CI) tests and to publish the modules to the IBM Cloud catalog. To update a Terraform repo to these requirements and add it to the terraform-ibm-modules GitHub project, follow these steps.
+
+For more information about how new modules are organized, see [module structure](module-structure.md).
+
+## Before you begin
+
+- Make sure that you meet the prerequisites in the [Before you begin](local-dev-setup.md#before-you-begin) section of "Local development setup" in the docs.
+
+## Create an empty module
+
+Create an empty module repo in the `terraform-ibm-modules` GitHub project.
+
+!> You likely do not have permissions to create a repo in the project. Submit an issue in the [terraform-ibm-issue-tracker](https://github.com/terraform-ibm-modules/terraform-ibm-issue-tracker/issues/new/choose) to request a repo.
+
+1.  In the upper right of any page in the [terraform-ibm-modules](https://github.com/terraform-ibm-modules) organization, use the **+** menu, and select **New repository**.
+1.  Select `terraform-ibm-modules` as the owner.
+1.  Enter a name for the module in format `terraform-ibm-<NAME>`, where `<NAME>` reflects the type of infrastructure that the module manages. Use hyphens as delimiters for names with multiple words (for example, terraform-ibm-`activity-tracker`).
+1.  Provide a short description of the module. The description is displayed under the repository title on the [organization page](https://github.com/terraform-ibm-modules) and in the **About** section of the repository. Use the description to help users understand what your repo does by looking at the description.
+1.  Confirm that you are creating a public repository in the terraform-ibm-modules organization and click **Create repository**.
+
+## Clone and initialize the repo
+
+Clone the repo that you just created.
+
+[inc-clone.md](inc-clone.md ':include')
```

Comment:
Looking at the PDF you attached, this section has a step to initialise the git submodule however if this is a blank repo, there won't be any git submodule added yet to initialise

![image](https://user-images.githubusercontent.com/29608224/182863660-d875044f-c983-411b-89ce-d1cfef5d37f5.png)

Reply:
ah Vincent already flagged this at https://github.com/terraform-ibm-modules/documentation/pull/12#discussion_r937822183

Reply:
OK. Put the updated instructions directly in the topic. No reuse now.